skbio/tree/_tree.py

```diff
@@ -2705,6 +2705,47 @@ def compare_tip_distances(self, other, sample=None, dist_f=distance_from_r,
         return dist_f(self_matrix, other_matrix)
 
     @experimental(as_of="0.4.0")
+    def bifurcate(self, insert_length=None, remove_singles=True):
+        r""" Restructures tree into a bifurcating tree
+
+        All nodes that have more than 2 children will
+        have additional intermediate nodes injected to ensure that
+        every node has only 2 children
```

Comment:
This is inaccurate in the case of a tree with an odd number of tips. It may be useful to distinguish "node" from "tip" as this method should ensure that _internal_ nodes have at most 2 children, but not that every node has at most 2 children.